src/oasis-indexer/api.ts

```diff
@@ -122,19 +92,20 @@ function arrayify<T>(arrayOrItem: null | undefined | T | T[]): T[] {
   return arrayOrItem
 }
 
-const _f1 = wrapWithNetwork(generated.useGetConsensusTransactions)
-
-export const useGetConsensusTransactions: typeof _f1 = (network, params?, options?) => {
+export const useGetConsensusTransactions: typeof generated.useGetConsensusTransactions = (
+  network,
+  params?,
+  options?,
+) => {
   const ticker = getTickerForNetwork(network)
-  return generated.useGetConsensusTransactions(params, {
+  return generated.useGetConsensusTransactions(network, params, {
     ...options,
     query: {
       ...options?.query,
       queryKey: [`/${network}/consensus/transactions`, ...(params ? [params] : [])],
     },
-    axios: {
-      ...options?.axios,
-      ...getBaseUrlParamFor(network),
+    request: {
+      ...options?.request,
       transformResponse: [
```

Comment:
Interesting, so is the `transformResponse` accepted both as an option inside `axios` and in `request` ?

Reply:
When customizing orval it changed the field name from `axios` to `request` (`query` field still doesn't accept it)